### ISSUE TITLE
bug:  Unknown command regex

### ISSUE THREAD

**Issue by wookayin** [altermo/ultimate-autopair.nvim#38]
I am using the out-of-box builtin config, and it happened when I type `"` in vimscripts.

Stacktrace:

```
stack traceback:
        ...d/ultimate-autopair.nvim/lua/ultimate-autopair/debug.lua:102: in function <...d/ultimate-autopair.nvim/lua/ultimate-autopair/debug.lua:100>
        [C]: in function 'error'
        ...autopair.nvim/lua/ultimate-autopair/extensions/rules.lua:50: in function 'check_rule'
        ...autopair.nvim/lua/ultimate-autopair/extensions/rules.lua:7: in function 'check_rule'
        ...autopair.nvim/lua/ultimate-autopair/extensions/rules.lua:55: in function 'check_rules'
        ...autopair.nvim/lua/ultimate-autopair/extensions/rules.lua:70: in function 'f'
        ...vim/lua/ultimate-autopair/configs/default/utils/init.lua:77: in function 'check'
        ...air.nvim/lua/ultimate-autopair/configs/default/paira.lua:97: in function <...air.nvim/lua/ultimate-autopair/configs/default/paira.lua:92>
        [C]: in function 'xpcall'
        ...d/ultimate-autopair.nvim/lua/ultimate-autopair/debug.lua:121: in function <...d/ultimate-autopair.nvim/lua/ultimate-autopair/debug.lua:120>
        ...ed/ultimate-autopair.nvim/lua/ultimate-autopair/core.lua:39: in function <...ed/ultimate-autopair.nvim/lua/ultimate-autopair/core.lua:33>
```

Debugging info:
```
{
  args = { {
      col = 1,
      incmd = false,
      key = '"',
      line = "",
      linenr = 3068,
      wcol = 1,
      wline = ""
    } },
  info = {
    _type = {
      [{}] = { "pair", "ambigous" }
    },
    backspace = <function 1>,
    check = <function 2>,
    conf = { '"', '"',
      rules = { { "when", { "filetype", "vim" }, { "not", { "regex", "^%s*$" } } } },
      string = true,
      suround = true
    },
    doc = 'autopairs ambigous pair: "',
    end_pair = '"',
    ext_checks = {},
    extensions = { {
        conf = {
          nofilter = false,
          other_char = { " " },
          p = 10
        },
        m = {
          call = <function 3>,
          check = <function 4>
        },
        name = "fly"
      }, {
        conf = {
          p = 20
        },
        m = {
          call = <function 5>
        },
        name = "suround"
      }, {
        conf = {
          p = 30
        },
        m = {
          call = <function 6>
        },
        name = "alpha"
      }, {
        conf = {
          p = 40
        },
        m = {
          call = <function 7>,
          check_rule = <function 8>,
          check_rules = <function 9>,
          init = <function 10>
        },
        name = "rules"
      }, {
        conf = {
          p = 60
        },
        m = {
          call = <function 11>,
          filter_out_string = <function 12>,
          filter_string = <function 13>,
          instring = <function 14>
        },
        name = "string"
      }, {
        conf = {
          filter = true,
          p = 70
        },
        m = {
          call = <function 15>
        },
        name = "escape"
      }, {
        conf = {
          nft = { "TelescopePrompt" },
          p = 80
        },
        m = {
          call = <function 16>
        },
        name = "filetype"
      }, {
        conf = {
          p = 90,
          types = { "/", "?", "@" }
        },
        m = {
          call = <function 17>
        },
        name = "cmdtype"
      } },
    fn = {
      check_end_pair = <function 18>,
      check_start_pair = <function 19>,
      find_end_pair = <function 20>,
      find_start_pair = <function 21>,
      in_pair = <function 22>,
      is_end = <function 23>,
      is_start = <function 24>
    },
    get_map = <function 25>,
    newline = <function 26>,
    p = 10,
    pair = '"',
    rule = <function 27>,
    sort = <function 28>,
    start_pair = '"'
  },
  mes = "...autopair.nvim/lua/ultimate-autopair/extensions/rules.lua:50: Unknown command regex"
}

```

**Comment by altermo** [comment]
Okay, update 0.6.0 where this is fixed.
Updating is a little more complicated as it requires a total reinstall:
+ Uninstall the plugin by removing it from the plugin list and cleaning
  + Note that if you are using a lockfile for plugins then it *might* save the old plugin branch
+ Install the plugin again

**Comment by wookayin** [comment]
Thanks for the fix, it works on v0.6.0. I think a mistake was made because the plugin does not use 'main' or 'master' as the main branch.

But out of curiosity, why is updating that complicated? Shoudln't it as easy as git-pull-ing the plugin? You don't have any build process for this plugin.

**Comment by altermo** [comment]
Each big release reworks like half the codebase, and I don't want to break others' configurations.
And the way I git-merge already breaks git-pull, so I don't know how to do it better.
